### ISSUE TITLE
Prism.Avalonia Design Pattern Support

### ISSUE THREAD

**Issue by DamianSuess** [AvaloniaUI/TemplateStudio#10]
### Is your feature request related to a problem? Please describe.

I would like to create an application that supports Prism.Avalonia

### Describe the solution you'd like

When creating a new application, I would like to be able to create one which uses Prism.Avalonia's framework for the MVVM design pattern.

### Describe alternatives you've considered

I've used ReactiveUI in the past, however, I prefer to use the Prism library toolset which comes with Prism.Avalonia such as its Region Manager and Journaling for page navigation, Modules, DialogService and etc.

### Additional context

Applies to platforms:
* Linux
* Windows
* macOS

### Applies to the following platforms:

- [ ] WinUI
- [ ] WPF
- [ ] UWP

**Comment by DamianSuess** [comment]
I was working on an extension to support this, however, I rather support the Avalonia ecosystem under one hood

**Comment by chenzanyu** [comment]
As a WPF developer, I am more familiar with Prism and would like to support Prism templates

**Comment by Coolkeke** [comment]
Highly recommended

**Comment by DamianSuess** [comment]
Gonna give this a shot in my spare time and create a PR

https://github.com/DamianSuess/TemplateStudio/tree/feature/PrismAvaloniaSupport

**Comment by carlosxjose** [comment]
I am wondering if the Prism design PR of DamienSuess it's going to be merge and release....

I wish you could get this possible... Thanks

**Comment by DamianSuess** [comment]
Yes, @carlosxjose, I'm still seeking assistance in building this project.
I will admit, I haven't had much time lately to commit to figuring out what wizardry needs to happen. Hopefully some time can be cleared up in the next few weeks.

**Comment by DamianSuess** [comment]
I'v created a template pack for [Prism.Avalonia](https://github.com/SuessLabs/Prism.Avalonia.Templates) which has been published to NuGet.org. Any chance this could be integrated into here?
